services/explorer/src/api/graphql/queries/pbftBlocks.ts

```diff
@@ -7,8 +7,24 @@ export const blockQuery = `
       gasLimit,
       gasUsed,
       timestamp,
+			parent {

```

Comment:
The spacing seems a bit weird. Is it like this in the code or just in the PR?

Reply:
Strange, it seems its just in the PR